src/Native/configure.cmake

```diff
@@ -235,10 +249,17 @@ check_cxx_source_runs(
 check_prototype_definition(
     getpriority
     "int getpriority(int which, int who)"
-    "0"
+    0
```

Comment:
Can we extract this change from this pull-request? I don't understand this change -- I would need to go for CMake documentation.